app/utils/metadata.py

```diff
@@ -2141,7 +2141,7 @@ def run(self) -> None:
                 dynamic_query.pfids_by_appid()
                 # Make sure we have PublishedFileIds to work with...
                 if (
-                    not len(dynamic_query.publishedfileids) > 0
+                    len(dynamic_query.publishedfileids) <= 0
```

Comment:
to simplify, it should probably just be `if len(dynamic_query.publishedfileids == 0` - i do get that this can be simplified though. It will never be less than 0.

Reply:
yeah right it will never be negitive so  i used your suggestion Thanks
if len(dynamic_query.publishedfileids == 0

Reply:
For safety its better to just check > since we don't have an unsigned type.

Though tbh I don't really see this as that big of a simplification from what it was before compared to what it is in this pr either way. Feels like semantics

Reply:
yeah sure